packages/component/src/Transcript/FocusTrap.tsx

```diff
@@ -49,11 +49,11 @@ const FocusTrap: FC<FocusTrapProps> = ({ children, onFocus, onLeave }) => {
 
   return (
     <Fragment>
-      <FocusRedirector onFocus={handleFirstSentinelFocus} />
+      {!android && <FocusRedirector onFocus={handleFirstSentinelFocus} />}
```

Comment:
How will it work in android in this case?

Reply:
and others where you put !android as a check

Reply:
This **FocusRedirector** component creating extra div with **tabindex="0"** to mange the focus state. When user on android devoice with talkback feature on it won't be creating those extra divs. I had tested this code with talkback feature enable and it is working as expected. When talk back is off the focus state is also getting managed in android devoice.